02/app.js

```diff
@@ -1 +1,13 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const element = document.querySelectorAll('a[data-url')
+
+element.forEach(takeElement => {
```

Comment:
Tutaj nie trzeba bo mamy wiele elementów (czyli będzie pusta tablica).

PS. Skoro to wiele elementów to nazwa zmiennej wprowadza nas w błąd :)